### ISSUE TITLE
Feature Request: Allow all the unit of measurement for speeds

### ISSUE THREAD

**Issue by GrazianoCapelli** [BasicAirData/GPSLogger#161]
To date the users can choose the unit of measurement for the speeds, but are limited to stay stick to the general units system set.
For example, when the "Metric" system is set, the units of measurement available for the speeds are "m/s" and "km/h"; there is no way to select, for example, "knots" or "mph".

Some users, that are using the app for sailing or for flights, would like to use mixed sets of units.
A classical example is - on sailing - the "Metric" system for distances and accuracies, but with speeds in "knots".

**Comment by GrazianoCapelli** [comment]
Should we continue to switch the speeds unit on "System of Measurement" change?

**Comment by GrazianoCapelli** [comment]
## TO DO
- [X] Implement the new Preference that allows all the speed units
- [ ] Convert the old unit of measurement for speeds into the new, and delete the old one
- [ ] Evaluate to implement some connection between "Units of Measurement" and "Speeds units"

**Comment by GrazianoCapelli** [comment]
We chosen to allow any unit of measurement for speed regardless of the general unit of measurement set.
Before that, when the user changed the general units of measurement, the speed unit changed accordingly.
The logical choice for me is to break the link between "Units of Measurement" and "Speed unit": from now on they must be set independently.

I made this decision considering also that, during these years of testing and use, I very rarely have changed the general unit of measurement (and mainly for testing purpose), but more often and for different purposes I have changed the set of speed units.

Anyway, let's start to collect feedback! in case, we'll link back them again.
